component/src/main/java/io/siddhi/extension/io/cdc/source/CDCSource.java

```diff
@@ -252,6 +258,15 @@
                         type = DataType.INT,
                         optional = true,
                         defaultValue = "-1"
+                ),
+                @Parameter(
+                        name = "cron.expression",
+                        description = "This is used to specify a timestamp in cron expression. " +
```

Comment:
I think the description is wrong. Please check

Reply:
fixed in ee838f87eb853dd4cccb33250c2fa8f7d620a3cc